docs/guides/validation/checkpoints/how-to-create-a-new-checkpoint.md

```diff
@@ -2,4 +2,103 @@
 title: How to create a new Checkpoint
 ---
 
-This article is a stub.
+This guide will help you create a new Checkpoint, which allows you to couple an Expectation Suite with a data set to validate.
+
+Note: As of Great Expectations version 0.13.7, we have updated and improved the Checkpoints feature. You can continue to use your existing legacy Checkpoint worfklows if you’re working with concepts from the Batch Kwargs (v2) API. If you’re using concepts from the BatchRequest (v3) API, please refer to the new Checkpoints guides.
+
+### Steps for Checkpoints (>=0.13.12)
+
+:::note Prerequisites: 
+
+This how-to guide assumes you have already:
+
+* [Set up a working deployment of Great Expectations](/docs/tutorials/getting-started/intro)
+* [Configured a Datasource using the BatchRequest (v3) API](/docs/tutorials/getting-started/connect-to-data)
+* [Created an Expectation Suite](/docs/tutorials/getting-started/create-your-first-expectations)
+
+:::
+
+##### 1. First, run the CLI command below.
+
+````console
+great_expectations --v3-api checkpoint new my_checkpoint
+````
+
+##### 2.  Next, you will be presented with a Jupyter Notebook which will guide you through the steps of creating a checkpoint.
+
+### Additional Notes
+
+Within this notebook, you will have the opportunity to create your own yaml Checkpoint configuration. The following text walks through an example.
+
+#### SimpleCheckpoint Example
+
+##### 2a. Here is a simple example configuration. 
+
+For this example, we’ll demonstrate using a basic Checkpoint configuration with the `SimpleCheckpoint` class, which takes care of some defaults. Replace all names such as `my_datasource` with the respective DataSource, DataConnector, DataAsset, and Expectation Suite names you have configured in your `great_expectations.yml`.
+
+````yaml
+config = """
+name: my_checkpoint
+config_version: 1
+class_name: SimpleCheckpoint
+validations:
+	- batch_request:
+datasource_name: my_datasource
+data_connector_name: my_data_connector
+data_asset_name: MyDataAsset
+data_connector_query:
+	index: -1
+expectation_suite_name: my_suite
+"""
+````
+
+This is the minimum required to configure a Checkpoint that will run the Expectation Suite `my_suite` against the data asset `MyDataAsset`. See [How to configure a new Checkpoint using test_yaml_config](/docs/guides/validation/checkpoints/how-to-configure-a-new-checkpoint-using-test_yaml_config) for advanced configuration options.
+
+##### 2b. Test your config using `context.test_yaml_config`.
+
+````python
+context.test_yaml_config(yaml_config=config)
+````
+
+When executed, test_yaml_config will instantiate the component and run through a self_check procedure to verify that the component works as expected.
+
+In the case of a Checkpoint, this means
+
+	1. validating the yaml configuration,
```

Comment:
This isn't rendering as a list for some reason. I believe the indentation is too much? Or L66 needs to be deleted I'm not sure.

Reply:
Done